pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -286,6 +286,7 @@ func (w *Webhook) inject(pod *corev1.Pod, _ string, _ dynamic.Interface) error {
 		return errors.New("cannot inject lib into nil pod")
 	}
 	injectApmTelemetryConfig(pod)
+	injectSecurityClientLibraryConfig(pod)
```

Comment:
I think we should move this somewhere under:
```
	if len(libsToInject) == 0 {
		return nil
	}
```

That way we'll only inject the security client envs if a lib is injected.